lang/en/docs/tutorials/other/jupyter.json

```diff
@@ -0,0 +1,180 @@
+{
+    "descriptionLinks": [
+        "Jupyter Notebook Environment: https://docs.exabyte.io/tutorials/other/jupyter"
+    ],
+    "description": "In this tutorial, we show how to create a Jupyter Notebook environment using our Web Interface.",
+    "tags": [
+        {
+            "...": "../../metadata/general.json#/tags"
+        }
```

Comment:
Let's add tags for data science and jupyter here too

Reply:
Done.

Reply:
We need separate tags for jupyter and data science. The latter has to go to the `data` folder.